addons/account/static/src/js/section_and_note_fields_backend.js

```diff
@@ -57,6 +57,12 @@ var SectionAndNoteListRenderer = ListRenderer.extend({
             $row.addClass('o_is_' + record.data.display_type);
         }
 
+        if (record.data && record.data.product_id
+            && record.data.product_id.data.display_name === "Down payment"
```

Comment:
@Feyensv pointed out to me that the display name is translated so you can't base your hack on it :/

Reply:
I saw that it is enough to verify the `invoice_status`, thank you ! :)